geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractHitsMissesIntegrationTest.java

```diff
@@ -244,28 +392,24 @@ public void testSunionstore() {
     runDiffStoreCommandAndAssertNoStatUpdates("set", (k, v, s) -> jedis.sunionstore(k, v, s));
   }
 
-  // ------------ Hash related commands -----------
-
+  // TODO our implementation updates the stats when it shouldn't
+  @Ignore
```

Comment:
I've opened GEODE-8857 to fix SMOVE. Please add this Jira to the `@Ignore` here.